ai_agents/openai_agent_researcher/workflow.py

```diff
@@ -0,0 +1,72 @@
+"""Common code to chat with the agent."""
+
+from os import getenv
+
+import data
+
+from ai import plan
+from ai import report
+from ai import search
+import slack
+from slack import ask
+from slack import send
+
+
+_prefix = f"!{getenv('INVOCATION_CMD', 'research')} "
+
+
+def workflow(q: str):
+    """Run the entire interaction with the user.
+
+    There are three phases:
+    1. Plan the search.
+    2. Execute the search.
+    3. Report the results.
+    """
+    # Plan the search.
+    search_plan = plan(q)
+
+    send("Now I will execute on the plan.\n")
+
```

Comment:
The empty lines here are making it hard to understand this block, not easier.
```suggestion
    send("Now I will execute on the plan.\n")
```

Reply:
done